test/src/test/java/lib/layout/ConfirmationLinkTest.java

```diff
@@ -89,9 +89,6 @@ private void checkRegularCasePost(TestRootAction testParams) throws Exception {
         testParams.paramPost = "true";
         assertMethodPostAfterClick();
     
-        testParams.paramPost = "TruE";
```

Comment:
This looks like a case we don't really need to support, @Wadeck to confirm as original author.

Reply:
✔️ No strict need, it's expected to be a boolean, I was just more lenient with the test.